recipes/kraken-ea/meta.yaml

```diff
@@ -0,0 +1,38 @@
+package:
+  name: kraken-ea
+  version: "0.10.5ea.3"
+source:
+  fn: v0.10.5-beta-ea.3.tar.gz
+  url: https://github.com/ExpressionAnalysis/kraken/archive/v0.10.5-beta-ea.3.tar.gz
+  md5: 34dce0103874e199a13eeed9bf20bd73 
+
+build:
+  number: 0
+  skip: True # [osx]
+  has_prefix_files:
+    - libexec/kraken
+    - libexec/kraken-build
+    - libexec/kraken-filter
+    - libexec/kraken-mpa-report
+    - libexec/kraken-report
+    - libexec/kraken-translate
+
+requirements:
+  build:
+    - perl
+    - zlib
+
+  run:
+    - zlib
+    - jellyfish 1.*
```

Comment:
I think the `*` notation is still reserved for special things like numpy. Does `jellyfish<2` work?